plasmapy/formulary/collisions.py

```diff
@@ -555,7 +558,9 @@ def _process_inputs(T: u.K, species: (particles.Particle, particles.Particle), V
     return T, masses, charges, reduced_mass, V
 
 
-def _replace_nan_velocity_with_thermal_velocity(V, T, m):
+def _replace_nan_velocity_with_thermal_velocity(
```

Comment:
Are you using this function anywhere anymore?

Reply:
This function is used in namurphy's functions for processing input. I just extended it to include particles other than electrons

Reply:
Ah, got it